src/vm.sh

```diff
@@ -3,6 +3,10 @@ include "$KW_LIB_DIR/kwlib.sh"
 
 function vm_mount()
 {
+  if [[ "$1" =~ -h|--help ]]; then
+    vm_help "$1"
+    exit 0
+  fi
```

Comment:
Why did you add this here? This does not look nice... the top of the function should be used to declare variables. If necessary, refactor this function to suit your goals better.
Ps.: Same for the parts that you have this.